Bindings/Java/OpenSimJNI/CMakeLists.txt

```diff
@@ -114,15 +114,18 @@ set_source_files_properties(
         PROPERTIES COMPILE_FLAGS "${COMPILE_FLAGS}")
 
 if(${OPENSIM_USE_INSTALL_RPATH})
```

Comment:
How would that behave on windows? Are we guaranteed OPENSIM_USE_INSTALL_RPATH is false in that case?